packages/backend/src/config/aiConfigSchema.ts

```diff
@@ -53,14 +36,13 @@ export const aiCopilotConfigSchema = z
                     apiKey: z.string(),
                     apiVersion: z.string(),
                     deploymentName: z.string(),
-                    temperature: z.number().min(0).max(2).default(0.2),
                 })
                 .optional(),
             anthropic: z
                 .object({
                     apiKey: z.string(),
                     modelName: z.string().default(DEFAULT_ANTHROPIC_MODEL_NAME),
-                    temperature: z.number().min(0).max(2).default(0.2),
+                    availableModels: z.array(z.string()).optional(),
```

Comment:
`modelName` is basically the active model name and availableModels is what will allow to be available from presets for the provider i guess